docs/mkdocs/docs/technical/architecture.md

```diff
@@ -8,18 +8,12 @@ Users interact with the C++ storage engine, and therefore the storage itself, vi
 
 Please note that there is **no required server** component.
 
-<figure markdown>
-  ![Image title](../../images/ArcticArchitecture.png)
-  <figcaption>ArcticDB Architecture</figcaption>
-</figure>
+![ArcticDB Architecture](../images/ArcticArchitecture.png)
```

Comment:
I agree there isn't much value add from `<figure>` at the moment, especially given the captions are never changed.

@mehertz What was the original intention?